packages/ui/.prettierrc.json

```
@@ -0,0 +1,11 @@
+{
```

my standard prettier config that i use across all projects (also aligns with this project)
```suggestion
{
        "semi": false,
        "singleQuote": true,
        "proseWrap": "never",
        "tabWidth": 4,
        "trailingComma": "all",
        "arrowParens": "avoid",
        "printWidth": 160,
        "endOfLine": "auto"
    }
```

---

Сможешь ли его вынести в корень проекта, чтобы я мог его активировать у себя и убрать этот?

---

i dont really understand what the problem is on your side...
unfortunately, i can't use prettier for the whole project because of its specifics e.g. in some files it makes code less readable